mod/01_core/spec/set/all/trash_spec.rb

```diff
@@ -11,5 +11,24 @@
       expect(Card[name]).to be
     end
   end
+  
+  it 'does not delete account with edits' do
+    Card::Auth.as_bot do
+      name = 'Joe User'
+      card = Card[name]
+      card.delete
+      expect(card.errors[:delete].first).to match("Edits have been made with #{name}'s user account")
+      expect(Card[name]).to be
+    end
+  end
+  
+  it 'deletes account without edits' do
+    Card::Auth.as_bot do
+      name = 'born to die'
+      card = Card.create! :name=>name, :type_code=>:user
+      card.delete
+      expect(Card[name]).not_to be
+    end
+  end
```

Comment:
Wait, but these are tests for the other PR, right?  I guess this branch also has the other branch in it?